lib/semantics/resolve-names.cc

```diff
@@ -4058,6 +4060,19 @@ ParamValue DeclarationVisitor::GetParamValue(const parser::TypeParamValue &x) {
       x.u);
 }
 
+ParamValue DeclarationVisitor::GetLenParamValue(
+    const parser::TypeParamValue &x) {
+  ParamValue param{GetParamValue(x)};
+  param.set_attr(common::TypeParamAttr::Len);
+  return param;
+}
+
+ParamValue DeclarationVisitor::GetLenParamValue(common::ConstantSubscript l) {
+  ParamValue param{l};
```

Comment:
A lowercase 'l' is not the most readable variable name.

Reply:
Yes, that was a bad name. I removed the function anyway based on Tim's comment.